dist/drawer-dialog/drawer-dialog.css

```diff
@@ -94,11 +91,13 @@
 .drawer-dialog__footer > :not(:first-child) {
   margin-left: 8px;
 }
-button.icon-button.drawer-dialog__close {
-  background-color: transparent;
+button.icon-btn.drawer-dialog__close {
+  align-self: start;
   border: 0;
-  height: auto;
-  outline-offset: -8px;
+  height: 32px;
+  min-width: 32px;
+  position: relative;
+  width: 32px;
```

Comment:
Why was this changed? 
I believe in drawer we allow putting in text possibly in the button, so this might break that. 
Again, this was done before icon button existed so just need to make sure this doesn't break that case.

Reply:
So this actually wasn't doing anything at all before, since `icon-button` should have been `icon-btn`. For these changes I pulled in all of our updates from lightbox dialog.